src/install/ibm-mq/intro.mdx

```diff
@@ -0,0 +1,34 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureIbmmqDashboard from 'images/infrastructure_screenshot-crop_ibmmq-dashboard.webp'
+
+<Callout title="Public preview">
+  This feature is currently a public preview.
+</Callout>
+
+Our IBM MQ [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations)
+collects and sends dimensional metrics from [IBM MQ](https://www.ibm.com/products/mq).
+
+You can view this [metric data](#metrics) in pre-built <InlinePopover type="dashboards" />, create <InlinePopover type="alerts" /> policies, and create custom queries and charts.
+
+<img
+  title="IBM MQ quickstart dashboard"
+  alt="Image of the dashboard available through the IBM MQ quickstart"
+  src={infrastructureIbmmqDashboard}
+/>
+
+<figcaption>
+  Dashboard installed through the [New Relic IBM MQ Monitor quickstart](https://one.newrelic.com/nr1-core?state=c95ed954-7277-e44d-7fd7-351a77b3ff37).
+</figcaption>
+
+To get data from IBM MQ infrastructure, our IBM MQ integration uses a third-party [Prometheus exporter](https://github.com/ibm-messaging/mq-metric-samples/tree/master/cmd/mq_prometheus), which exposes Prometheus metrics on the port specified in the configuration. The integration collects these metrics, transforms them into entities, filters them, and then sends them to New Relic.
```

Comment:
Not 100% sure, but "transforms them into entities" doesn't seem right. The specific infra being monitored by this integration install would be the entity.

I'd just cut out the "transforms them into entities"